src/components/DesmosPostHogProvider/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { PostHogProvider } from 'posthog-react-native';
+import { useSetting } from '@recoil/settings';
+// eslint-disable-next-line import/no-unresolved
+import { POSTHOG_API_KEY } from '@env';
+
+export interface Props {
+  children: React.ReactNode;
+}
+
+const DesmosPostHogProvider: React.FC<Props> = ({ children }) => {
+  const analyticsEnabled = useSetting('analyticsEnabled');
+
+  return (
+    <PostHogProvider
+      children={children}
+      apiKey={POSTHOG_API_KEY}
+      options={{
+        host: 'https://app.posthog.com',
+        enable: analyticsEnabled,
+      }}
+      autocapture={{
+        captureLifecycleEvents: false,
+        captureTouches: true,
```

Comment:
I think it's better to disable this since we might generate too many events and we have a 1 million events/month limit on the free tier. Also I don't think touches on all views are worth. Any thoughts on this @bragaz?

Reply:
I had a call before with @ale-mazz that explained to me the same concern that you have. I didn't think that the auto capture was getting so many events, from the docs I thought it was only capturing a few things, not following the user everywhere. I honestly think that it's too much. Integrating the custom events I've prepared is enough at the moment.

Reply:
@bragaz @RiccardoM I removed the autocapture events.